.github/ISSUE_TEMPLATE/custom.md

```diff
@@ -0,0 +1,10 @@
+---
+name: Custom issue template
+about: Describe this issue template's purpose here.
```

Comment:
this template looks wrong to me

Reply:
Yeah updated in master

Reply:
Quickly used the web UI to get basic stuff setup